src/components/Card/Card.js

```diff
@@ -2,30 +2,32 @@ import React from 'react';
 import './Card.scss';
 
 const Card = (props) => {
-// For movie cards the props being passed down will be 
+// For movie cards the props being passed down will be
 
 
 // For character cards the props being spread out and passed down will need to be grabbed
 // as {id, name, homeworld, species}
 
   // homeworld, population of homeworld is a url for another fetch request
-  // species is another fetch request 
+  // species is another fetch request
   // films is another fetch request
     return (
-        <section className="section-card">
-          <div className="wrapper-card-div">
-            <ul className="card-details-ul">
-              <li>Episode {props.id}: {props.title}</li>
-              <li>Release Year: {props.date}</li>
-            </ul>
-            <button>View Characters</button>
-          </div>
-        </section>
+      <section className="section-card">
+        <div className="wrapper-card-div">
+          <ul className="card-details-ul">
+            <li><strong>Name:</strong> Luke SkyWalker</li>
+            <li><strong>Homeworld:</strong> Tattoiine</li>
+            <li><strong>Population of Tatootine:</strong> 400 million</li>
+            <li><strong>Species:</strong> Human</li>
+            <li><strong>Other Films:</strong> all seven</li>
+          </ul>
+        </div>
+      </section>
     )
   }
```

Comment:
Hey it looks like this is overwriting the changes I made last night to get the props to display the fetched movie data.

Reply:
@JamesRexMiller4 Sorry about that, I meant to paste that in the commented out section. I have reset it back to what you wrote. I think we will have to alter this a bit to get my styling back but I'll work on the redirection first.